mycli/packages/special/iocommands.py

```diff
@@ -12,6 +13,7 @@
 from .main import special_command, NO_QUERY, PARSED_QUERY
 from .favoritequeries import favoritequeries
 from .utils import handle_cd_command
+from ..prompt_utils import confirm_destructive_query
```

Comment:
I know some of the existing imports are relative, but especially with the double-dot import, can we do an absolute import?

Reply:
Of course, I did it only to follow the existing convention, I'll fix it in the next commit.